src/hw_isolation_record/manager.cpp

```diff
@@ -652,6 +672,13 @@ void Manager::updateEntryForRecord(const openpower_guard::GuardRecord& record,
         updated = true;
     }
 
+    // TODO:https://github.com/ibm-openbmc/openpower-hw-isolation/issues/39
+    // During "core" checkstop PLDM will be blocked on the DMA transfer of
+    // the dump data and might not honor enabling resolved D-Bus property
+	// of core D-Bus object hosted by PLDM during isolation entry creation.
+	// To keep the system consistent setting enabled property here.
+	utils::setEnabledProperty(_bus, *isolatedHwInventoryPath, false);
```

Comment:
What happens if the PLDM hangs in this flow too? I think we should handle that case wherever using the "setEnabledProperty".

I suggest adding a code to handle that exception case inside setEnabledProperty with a comment like you did here.

Reply:
nothing serious anyhow the guard record will be already there just that some properties are not up to date.